src/main/java/seedu/address/logic/commands/FilterInterviewTimeCommand.java

```diff
@@ -26,10 +26,12 @@ public class FilterInterviewTimeCommand extends FilterCommand {
             + "Example:\n"
             + "- " + COMMAND_WORD + "before/020220241100 after/020220251100 from/030420240800-020520241100";
     public static final String WRONG_INTERVIEW_TIME_RANGE_MESSAGE = "INTERVIEW_TIME_RANGE should be in the form of:\n"
-            + "before/INTERVIEW_TIME\nor\n"
-            + "after/INTERVIEW_TIME\nor\n"
-            + "from/INTERVIEW_TIME-INTERVIEW_TIME\n"
-            + "where INTERVIEW_TIME is in the form: " + InterviewTime.INTERVIEW_TIME_FORMAT
+            + "- before/INTERVIEW_TIME\n"
+            + "- after/INTERVIEW_TIME\n"
+            + "- from/INTERVIEW_TIME-INTERVIEW_TIME\n"
+            + "where there are no white spaces before and after the '-' "
```

Comment:
a suggestion would be `before and after the hyphen` or `before and after '-'`, same goes for other references to '-'

Reply:
ok I will use `before and after '-'`